### PR TITLE
Allow init container to set kernel core pattern

### PR DESCRIPTION
**Pull request by andraxylia**
Fixes https://github.com/istio/manager/issues/364

---
This change is [Reviewable](https://reviewable.io/reviews/istio/istio/71)

### CODE REVIEW

File: demos/apps/bookinfo/bookinfo-istio.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ spec:
             "name": "iptables",
             "image": "istio/init:2017-03-17-22.11.25",
             "imagePullPolicy": "Always",
-            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] } }
+            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] }, "privileged": true }
```
Comment (kyessenov) [inline demos/apps/bookinfo/bookinfo-istio.yaml]: We should be careful what permissions we get in the container. Do we really need full privileges? If so, you should remove capabilities.

**Comment by andraxylia** [comment]
This is the only way that (I know) it works. I am open to suggestions. While this is still in development, it's a compromise we can live with. I will remove capabilities.

**Comment by kyessenov** [comment]
Consider adding a start script to debug container that sets the core pattern instead.

**Comment by andraxylia** [comment]
Nothing will work unless is privileged.

**Comment by kyessenov** [comment]
This is done through injection script.

**Pull request closed without merging**